text/fuzz/corpus/from_test_parse_error_0

```diff
@@ -0,0 +1 @@
+bla 3.14
```

Comment:
All the `from_test_parse_` are written out as a one-time job from the test-suite, so it should be a clone of whatever was in there.